### PR TITLE
Add 20241222 Playtest announcement

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: news/_posts/2024-12-22-playtest-20241222.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+kind: article
+title: "Playtest 20241222"
+author: "PunkPun"
+created_at: 2024-12-22 22:00 +0000
+disqus_id: "playtest-20241116"
```
Comment (penev92) [inline news/_posts/2024-12-22-playtest-20241222.md:6]: Is this supposed to be using the old ID?

File: news/_posts/2024-12-22-playtest-20241222.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+permalink: "/news/playtest-20241222/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20241222]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/INSERTHERE):
+
+Notable changes include:
+* saboteurs can now cloak while moving
```
Comment (penev92) [inline news/_posts/2024-12-22-playtest-20241222.md:16]:
```suggestion
* Saboteurs can now cloak while moving
```

File: news/_posts/2024-12-22-playtest-20241222.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* added delete area button to the map editor
+* improved Battlefield News dropdown
+* improved formatting of unit descriptions and encyclopedias
+* fixed Dune 2K not detecting installed music and videos
```
Comment (penev92) [inline news/_posts/2024-12-22-playtest-20241222.md:20]: Both "d2k"/"D2k" and "Dune 2000" are widely used, but this version is very rare. Can we go with "Dune 2000" here?

**Review by penev92 — COMMENTED** [review]
Listing features in `20241116` used '-' and here it uses '*' ?
Ofc all the versions will need to be updated (hopefully to `20241228`).

P.S.: Since you're running the website locally anyway, can you provide a screenshot on the PR please?

File: news/_posts/2024-12-22-playtest-20241222.md (hunk 1 above)
Comment (PunkPun) [reply news/_posts/2024-12-22-playtest-20241222.md:6]: yes, we retain the disquss link

**Comment by PunkPun** [comment]
Screenshot_2024-12-27_at_18 57 49

**Pull request merged**